### ISSUE TITLE
Too many clicks to get to submenu in responsive

### ISSUE THREAD

**Issue by emmairwin** [emmairwin/community-edu#26]
We need fewer actions for sub menu in responsive which currently are:

1) Click on icon which expands menu
2)  Click on parent item  (reloads responsive view)
3) Click on icon which expands menu (now we finally see sub menu)
4) Click on sub menu.

Suggesting

1> Click on icon which expands menu
2. If top menu item was clicked, automatically display sub menu on page load.
3. User can click on sub menu
Basically this skips the double-responsive collapse scenerio.

**Comment by asdofindia** [comment]
Do we need to make the top links clickable? They're causing all the problems. If we get rid of that restriction, we can have the normal bootstrap dropdowns working beautifully.

Also, we can add an "Introduction" link in the dropdown that will do the work that the top menu items do now.

**Comment by emmairwin** [comment]
@asdofindia the quick answer is no - we don't need them to be clickable.  As long as people can click the submenu in mobile , then totally fine with other options.  To be warned though the hover event translates as a click in Android (which is what started the problem in the first place).  It needs to work in Android.
